### PR TITLE
Adding tutorial describing steps to protect OpenStack workloads using neutron sfc

### PR DESCRIPTION
**Pull request by pravinrajr9**
Adding tutorials to protect OpenStack using OSC and OSC Neutron NSFC plugin. Provides step by step information to use OSC to protect OpenStack workloads using Neutron SFC functionality

### CODE REVIEW

**Comment by pravinrajr9** [comment]
@sahasamiran @arvindn05 please review

**Review by arvindn05 — COMMENTED** [review]
add how you would create a chain of services.

File: tutorials/openstack_workload_sfc.md
Comment (pravinrajr9) [reply tutorials/openstack_workload_sfc.md]: line no 80 and 81, I dont know the reason to remove, Since I have mentioned NSFC plugin and Neutron SFC as SDN component.

File: tutorials/openstack_workload_sfc.md
Comment (pravinrajr9) [reply tutorials/openstack_workload_sfc.md]: done

File: tutorials/openstack_workload_sfc.md
Comment (pravinrajr9) [reply tutorials/openstack_workload_sfc.md]: done

**Comment by pravinrajr9** [comment]
@arvindn05 made few changes, please review

File: tutorials/openstack_workload_sfc.md
Comment (arvindn05) [inline tutorials/openstack_workload_sfc.md]: sytemids -> system id's

File: tutorials/openstack_workload_sfc.md
Comment (arvindn05) [inline tutorials/openstack_workload_sfc.md]: might be better to add links like

### 3. Define Manager Connector -> link to (openstack_workload.md#3-define-manager-connector) etc

**Review by arvindn05 — CHANGES_REQUESTED** [review]

**Review by sahasamiran — CHANGES_REQUESTED** [review]
please do the changes.

File: tutorials/openstack_workload_sfc.md
Comment (sahasamiran) [inline tutorials/openstack_workload_sfc.md]: Fix the position of the line. it should come as next line after the image.

**Review by sahasamiran — CHANGES_REQUESTED** [review]

**Review by sahasamiran — DISMISSED** [review]

**Comment by sahasamiran** [comment]
@arvindn05 waiting for your review. Once done will merge this patch.

File: tutorials/openstack_workload_sfc.md
Comment (arvindn05) [inline tutorials/openstack_workload_sfc.md]: typo

File: tutorials/openstack_workload_sfc.md
Comment (arvindn05) [inline tutorials/openstack_workload_sfc.md]: add  a note mentioning for a chain multiple vs ids can be provided in the order required.

File: tutorials/openstack_workload_sfc.md
Comment (arvindn05) [inline tutorials/openstack_workload_sfc.md]: screenshot should show the vcid,sgid parameters filled in and matching the json body

File: tutorials/openstack_workload_sfc.md
Comment (arvindn05) [inline tutorials/openstack_workload_sfc.md]: explain how policy id's are specified, for a single and a chain

File: tutorials/openstack_workload_sfc.md
Comment (arvindn05) [inline tutorials/openstack_workload_sfc.md]: i think we can eliminate this section ad incorporate this info into the Define a Service Function Chain section.

**Review by arvindn05 — DISMISSED** [review]

**Comment by arvindn05** [comment]
@sahasamiran @pravinrajr9 please go ahead and make the changes requested and checkin after your mutual reviews.

File: tutorials/openstack_workload_sfc.md
Comment (pravinrajr9) [reply tutorials/openstack_workload_sfc.md]: @arvindn05 Creation of multiple service function chain requires some steps like creation of DA, DS, importing another sensor image and binding. so for binding again the user need to refer step 9 , one has go back and forth for referring few steps so

File: tutorials/openstack_workload_sfc.md
Comment (pravinrajr9) [reply tutorials/openstack_workload_sfc.md]: done

File: tutorials/openstack_workload_sfc.md
Comment (pravinrajr9) [reply tutorials/openstack_workload_sfc.md]: updated in chain with multiple sf section

File: tutorials/openstack_workload_sfc.md
Comment (pravinrajr9) [reply tutorials/openstack_workload_sfc.md]: done

File: tutorials/openstack_workload_sfc.md
Comment (pravinrajr9) [reply tutorials/openstack_workload_sfc.md]: updated in chain with multiple sf section

**Review by sahasamiran — APPROVED** [review]

**Pull request merged**
